crates/sncast/src/lib.rs

```diff
@@ -65,7 +65,8 @@ impl FromStr for AccountType {
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         match s {
             "open_zeppelin" | "open-zeppelin" | "oz" => Ok(AccountType::OpenZeppelin),
-            "argent" => Ok(AccountType::Argent),
+            // We still accept "argent" for backward compatibility
+            "argent" | "ready" => Ok(AccountType::Ready),
```

Comment:
I think we should show some deprecation warning if user is using `argent`

Reply:
Yes, that's what I've mentioned in https://github.com/foundry-rs/starknet-foundry/pull/3504#discussion_r2177543272

Reply:
I think the warning should show up if they pass these as command line arguments. Don't think we should be showing warnings about file contents.